ionos_prd/issuer/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: issuer
 description: Chart holder for ArgoCD  
 type: application
-version: 1.0.18
+version: 1.0.19
```

Comment:
I can see that for DEV2 you have used version: 1.0.20 here, please keep consistency and fix one of the values.

Reply:
I fixed the value